docs/using-scylla/cassandra-compatibility.rst

```diff
@@ -30,9 +30,6 @@ Interfaces
      - | Fully compatible with version 3.3.1, with additional features from later CQL versions (for example, :ref:`Duration type <durations>`).
        | Fully compatible with protocol v4, with additional features from v5.
      - More below
-   * - Thrift 
-     - Deprecated in ScyllaDB and Cassandra
-     - Support for the Thrift protocol is deprecated and will be dropped in future releases of ScyllaDB. 
```

Comment:
Shouldn't we continue saying that Thirft existed in Cassandra 3 and we no longer support it, so it's a compatibility difference?

Reply:
indeed. added it back with more precise information.